ci/official/README.md

```diff
@@ -1,295 +1,131 @@
-each script reads its settings from the file denoted by the `TFCI` environment
-variable. All settings are prepended by `TFCI_`. Executing a build script looks
-like this:
-
-```
-cd <tensorflow-root-directory>
-mkdir -p build_output
-cp ci/official/envs/sample build_output/env
-vim build_output/env  # update "your_choice_here" to a real path
-export TFCI=$(realpath build_output/env)
-./ci/official/wheel.sh
-ls build_output
-```
-
-The scripts are intended to be easy to use for both CI systems and for local
-replication.
-
-### The Important TensorFlow Test Scripts
-
-Generally speaking, changes to TensorFlow are gated by these test scripts:
-
-1.  `wheel.sh` builds the TensorFlow Pip package and verifies its contents.
-2.  `pycpp.sh` runs an extensive `bazel test` suite whose targets vary depending
-    on the platform (target selection is handled in TensorFlow's bazelrc)
-3.  `libtensorflow.sh` builds
-    [libtensorflow](https://www.tensorflow.org/install/lang_c).
-4.  `code_check_full.sh` and `code_check_changed_files.sh` run some static code
-    analysis checks.
-
-Our CI runs these under a variety of environments that will receive additional
-documentation in the future.
-
-### Running Tests Yourself
-
-To run tests yourself, you'll copy the `envs/sample` file, adjust it to match
-your environment, `export TFCI=your-path`, and then simply run the script you
-want. A complete example is below this explanation. Some tips:
-
--   We generally use `<tensorflow-directory>/build_output` for all temporary
-    storage and build artifacts, and you'll find all output files there,
-    including `script.log`, the log of the last executed build script.
--   You can find out which env file a TensorFlow job used by looking at the
-    `TFCI` variable in the `BUILD_CONFIG` section of the Invocation Details for
-    that job, either in Sponge (internal) or ResultStore (external).
--   The files in `envs/` are configured to match TensorFlow's CI system and
-    reference paths and settings. They will not work out-of-the-box, so you'll
-    need to copy `sample` instead, which removes all of those custom details.
--   `sample` also resets the Python version to TensorFlow's default. You can
-    target a specific version by providing e.g.
-    `--repo_env=TF_PYTHON_VERSION=3.10` in `TFCI_BAZEL_COMMON_ARGS` as the other
-    `env` files do.
-
-Here is a complete example of how to set up and run a script:
-
-```
-cd <tensorflow-root-directory>
-mkdir -p build_output
-cp ci/official/envs/sample build_output/env
-vim build_output/env  # update "your_choice_here" to a real path
-export TFCI=$(realpath build_output/env)
-./ci/official/wheel.sh
+## TensorFlow's Official CI and Build/Test Scripts
+
+TensorFlow's official CI jobs run the scripts in this folder. Our internal CI
+system, Kokoro, schedules our CI jobs by combining a build script with an `env` file that is filled with configuration options:
```

Comment:
Hmm, maybe worth stating that the env files correspond to the envs folder in this directory.  I know most people will notice but i could see a few asking "What env files"?

Reply:
Good idea, tweaked the wording here.